src/CloudFlare/Zone/Settings.php

```diff
@@ -471,6 +471,8 @@ public function change_ipv6($zone_identifier, $value = 'off') {
 	 * @return array|mixed
 	 */
 	public function change_minify($zone_identifier, $value) {
+		
+		$data = ['value' => $value];
```

Comment:
Definitely! Didn't know if there was a reason "settings" wasn't done that way. Didn't want to muck with it if you had done it on purpose.